librz/core/cmd/cmd_linux_heap_slub.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2021 Pulak Malhotra <pulakmalhotra2000@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_types.h>
+#include <rz_core.h>
+#include <rz_heap_glibc.h>
+#include "cmd_descs.h"
+#include "../linux_heap_slub.h"
+#include "../linux_heap_slub64.h"
+
+#define call_handler(fun, ...) \
+	{ \
+		if (core->rasm->bits == 64) { \
+			return fun##_64(core, ##__VA_ARGS__); \
+		} else { \
+			return fun##_32(core, ##__VA_ARGS__); \
+		} \
+	}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_slub_dump_freelist_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *output_state) {
+	call_handler(rz_cmd_debug_slub_dump_freelist_handler, argc, argv, output_state);
+}
```

Comment:
Maybe I'm wrong but can't we just do the following?
```suggestion
RZ_IPI RzCmdStatus rz_cmd_debug_slub_dump_freelist_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *output_state) {
        if (core->rasm->bits == 64) {
                return rz_cmd_debug_slub_dump_freelist_handler_64(core, argc, argv, output_state);
        } else {
                return rz_cmd_debug_slub_dump_freelist_handler_32(core, argc, argv, output_state);
        }
}
```

Reply:
I will add more functions. Macro will come in handy. 
See [librz/core/cmd/cmd_linux_heap_glibc.c](https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd/cmd_linux_heap_glibc.c)